README.md

```diff
@@ -1 +1,11 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# My name is Alia Ai-Ming Tehzeeb Shahzad. 
+
+I'm a **third year** at the College.
+
+I'm double-majoring in *public policy* and *creative writing* inshallah. 
+
+[This](https://www.youtube.com/watch?v=7D3Ud2JIFhA) is my favorite album. 
+
+![This](https://3d4igz27oxtl2iwox73y9smh-wpengine.netdna-ssl.com/wp-content/uploads/2018/07/scottie-dogs-panting.jpg) is the dog I want when I grow up.
```

Comment:
I also had some trouble with inserting an image, so props to you for making it come out right! (LOVE this dog btw)